MemberExpectations.md

```diff
@@ -0,0 +1,60 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+It is understood that members will have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
+discourse is encouraged internally, members also have the obligation outside
+of the organization (for example in social media) to be respectful of the
+decisions made within the projects's documented processes even (and especially)
+if the results run counter to their person point of view.  A member who does
+not support a team decision or current team value may express the member's
+opposition within the team in an appropriate manner, but must be extremely
+careful when expressing the opinion externally.  External communication must
+not be done with the intent of undermining a decision, project values, or
+actions of the team. If another team member expresses that they believe external
+communications appear to have the intent to undermine the committee
+it is the responsibility of the individual expressing their opinions to rectify
+the situation immediately. If a member feels that there is no recourse aside
+from going public to campain against the teams decision on an extremely
+contentious issue then they should resign from the team before doing so.
```

Comment:
This entire paragraph comes across fairly negative to me. I'd prefer a different approach...

```
All participation in the Node.js project is voluntary, with each individual contributor having
their own reasons and motivations for wanting to contribute. Individual opinions and points
of view are natural and are to be expected. All contributors are expected to not only be
respectful of such differences, but to actively seek to work collaboratively with one another
in a manner that constructively elevates all contributors.

When a individual contributor chooses to accept an invitation to participate more actively
within the project either as part of a working group, the TSC, or Community Committee,
they implicitly take on an additional obligation to protect and be respectful of both the
project and all other contributors.

When decisions are made within the established guidelines and policies of the project,
contributors who have chosen to participate in leadership roles have a responsibility to
uphold and respect the decision even if they disagree with it. Should the member be
unwilling or unable to do so, then they should step down and resign their position as a
contributor to the project.
```

Reply:
That updated text sounds reasonable to me, will incorporate in next pass.